packages/terra-form-input/src/InputField.jsx

```diff
@@ -137,6 +144,7 @@ const InputField = (props) => {
       isInline={isInline}
       isLabelHidden={isLabelHidden}
       htmlFor={inputId}
+      style={styles}
```

Comment:
The Field has a class applying a maxWidth, this will overwrite the class style. This might be something to check with UX on to make sure it's okay to extend beyond the maxWidth set in the Field itself.

https://github.com/cerner/terra-core/blob/50a0e01af972252c657d020790e1b0f94dc02be4/packages/terra-form-field/src/Field.module.scss#L5

Reply:
Yeah, I think the behavior of having `style={{maxWidth: 100%}}` or `maxWidth="100%" will override the default set by the class is fine.